### PR TITLE
[ADD] LocalStorage

### PR DESCRIPTION
**Pull request by flyingrub**
Split the player in 3 parts : the play buttons ; the timeline ; the volume / repeat button
It allows to rerender more smartly the views.

Also added support of localStorage :  repeat, currentVolume and mute state are stored in it. https://github.com/cozy-labs/cozy-music/issues/59

### CODE REVIEW

File: app/libs/utils.js [hunk 1]
```diff
@@ -14,3 +14,15 @@ export function timeToString(time) {
             + seconds;
     return r;
 };
+
+export function hasLocalStorage() {
```
Comment (m4dz) [inline app/libs/utils.js]: I think you can safely ommit this test feature, as czy target browsers (last versions only) supports all `localStorage`: http://caniuse.com/#search=localstorage (expect OperaMini which, in fact, isn't a browser and can't be used to access Cozy).

File: app/views/app_layout.js [hunk 2]
```diff
@@ -11,18 +13,33 @@ const AppLayout = Mn.LayoutView.extend({
...
     },
 
     onRender() {
+        application.audio = this.ui.player.get(0);
         this.showChildView('content', new ContentView({
             model: application.appState
         }));
-        this.showChildView('player', new PlayerView());
         this.showChildView('toolbar', new ToolbarView());
+                this.showChildView('playerControls', new PlayerControlsView({
```
Comment (m4dz) [inline app/views/app_layout.js]: Any indent issue?

**Comment by m4dz** [comment]
Nice, good work! Does the `player.js` view is still relevant or can it be removed too?

**Comment by flyingrub** [comment]
Updated :)

File: app/models/appState.js [hunk 3]
```diff
@@ -1,18 +1,45 @@
 import Backbone from 'backbone';
+import { hasLocalStorage } from '../libs/utils';
```
Comment (m4dz) [inline app/models/appState.js]: This util has now disapear, nope?

Comment (flyingrub) [inline app/models/appState.js]: correct !

**Comment by m4dz** [comment]
Awesome, thanks :tada:

**Pull request merged**